OpenUtau.Core/Render/RenderEngine.cs

```diff
@@ -78,19 +78,20 @@ public class Progress {
             }
             items = items.OrderBy(item => item.PosMs).ToList();
             int threads = Util.Preferences.Default.PrerenderThreads;
-            var progress = new Progress(items.Count);
             var task = Task.Run(() => {
-                var progress = new Progress(items.Count);
-                Parallel.ForEach(source: items, parallelOptions: new ParallelOptions() {
-                    MaxDegreeOfParallelism = threads
-                }, body: item => {
-                    if (source.Token.IsCancellationRequested) {
-                        return;
-                    }
-                    item.progress = progress;
-                    Resample(item);
-                });
-                progress.Clear();
+                if (items.Count > 0) { 
```

Comment:
"phonemizer is initializing" was overlapped by this "Clear" even if there was no render items

Reply:
Please prefer returning early, to reduce nesting i.e.
```
if (item.Count == 0) {
  return;
}
```